styles/index.css

```
@@ -179,4 +180,207 @@ footer > div {
 .footer-sns img {
     height: 20px;
     vertical-align: top;
+}
+
+/* PC & Tablet */
+@media (min-width: 768px) {
+    header #logo-small {
+        height: 0;
+    }
```

❗️ 수정요청
로고가 화면 크기별로 두종류가 있어 판다얼굴만 있는 로고는 보여질 필요가 없어 height 0을 해주신 것 같아요!
이러한 경우 display none이 더 직관적이니 아래처럼 수정하시는 것을 추천드려요!

```suggestion
    header #logo-small {
        display: none;
    }
```